README.md

```diff
@@ -29,6 +29,10 @@
 * https://github.com/vaxxnz/nzcp-js
 * https://github.com/servian/nz-covidpass-verification-api
 
+### iOS SDK
```

Comment:
I wasn't sure if there was a particular ordering to the validator list? 
It seemed like a native iOS swift validator that can be plugged into many other mobile apps might be quite useful, so I didn't place it at the bottom, but if the maintainers would prefer we move this down the list, that's no problem